packages/liveblocks-core/src/client.ts

```diff
@@ -289,15 +289,16 @@ export type ClientOptions<TUserMeta extends BaseUserMeta = BaseUserMeta> = {
 //     | ((room: string) => Promise<{ token: string }>);
 //
 
-function getBaseUrlFromClientOptions(clientOptions: ClientOptions) {
-  if ("liveblocksServer" in clientOptions) {
-    throw new Error("Client option no longer supported");
-  }
+function getBaseUrl(baseUrl?: string | undefined): string {
+  baseUrl ||=
+    process.env.LIVEBLOCKS_BASE_URL ||
+    process.env.NEXT_PUBLIC_LIVEBLOCKS_BASE_URL ||
```

Comment:
We could also add `VITE_LIVEBLOCKS_BASE_URL` to be exhaustive and cover the remaining examples (Vite covers non-Next.js React examples but also Vue, Svelte, etc).

Reply:
Great point. Added in ad2e1c5261aabd652056cdad1286cf097886797d 👍